### ISSUE TITLE
v2 aws kafka create-configuration behavior is different than v1

### ISSUE THREAD

**Issue by herbertgoto** [aws/aws-cli#5001]
With v1 of the awscli I'm able to make a command call like:

`aws kafka create-configuration --name "Test2" --kafka-versions "2.3.1" --server-properties file://configuration.txt`

However, with v2, running the same command the awscli returns an error of:

```
An error occurred (BadRequestException) when calling the CreateConfiguration operation: [ {
  "name" : "j�hr����h�',zv���k��^��^��bq��nW",
  "value" : "",
  "errorMessages" : [ "Unsupported config j�hr����h�',zv���k��^��^��bq��nW for Kafka Version 2.3.1" ]
} ]
```

aws-cli/2.0.0 Python/3.7.4 Darwin/18.7.0 botocore/2.0.0dev4

**Comment by herbertgoto** [comment]
BTW, my configuration file has following content:

```
auto.create.topics.enable = true
delete.topic.enable = true
log.retention.hours = 8
```

**Comment by KaibaLopez** [comment]
Hi @herbertgoto ,
Yea, I get the same error, thanks for pointing it out!

**Comment by stealthycoin** [comment]
This is a blob binary type input structure, which means it is affected by our change to how binary input and output work in the CLI. Upgrading guide is here: https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration.html#cliv2-migration-binaryparam

Basically you have two options, revert to previous behavior with a config option(outlined in the link). Or use `fileb://` instead of `file://`.

**Comment by herbertgoto** [comment]
You are right. Nevertheless, the documentation does not state it like that. 
![Screen Shot 2020-02-28 at 4 30 14 PM](https://user-images.githubusercontent.com/8704017/75589274-577a4f00-5a48-11ea-9934-94551a069df2.png)

**Comment by stealthycoin** [comment]
Merged #6551
